tmk_core/common/avr/suspend.c

```diff
@@ -163,6 +163,10 @@ void suspend_power_down(void) {
     rgblight_suspend();
 #    endif
 
+#    if defined(LED_MATRIX_ENABLE)
+    led_matrix_set_suspend_state(true);
+#    endif
+
```

Comment:
```suggestion
```

Reply:
The spacing is consistent with how the file was. But ...